### PR TITLE
Publish Henk's chef-solo tip

### PR DESCRIPTION
**Pull request by adrianmoisey**

### CODE REVIEW

File: _posts/2015-03-12-chef-solo-your-env.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    done
+}
+
+function chefmyenv() {
+    ENV=${1:-default.env.name.here}
+    rolestojson
+    cat >! ${CHEF_DIR}/solo.rb <<EOF
+role_path "/home/user/chef/roles"
+environment_path "/home/user/chef/environments"
+cookbook_path ["/home/user/chef/cookbooks", "/home/henk/chef/external-cookbooks"]
```
Comment (stefanor) [inline _posts/2015-03-12-chef-solo-your-env.md:38]: one user, one henk

**Comment by imorrison** [comment]
OMG a new script tip: https://yola.github.io/script-tip/2015/03/12/chef-solo-your-env.html

---

I suppose I should add some css...

**Comment by stefanor** [comment]
Is there an RSS feed?

**Comment by stormf** [comment]
I want to can this and just add it to the chef repo as a sh script

**Pull request merged**